rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalTextDocumentService.java

```diff
@@ -488,4 +490,14 @@ private static <T> CompletableFuture<T> recoverExceptions(CompletableFuture<T> f
                     return defaultValue.get();
                 });
     }
+
+    @Override
+    public boolean isManagingFile(ISourceLocation file) {
+        return facts.containsFile(file);
```

Comment:
why is this not looking at documents? (just like the Parametric one, where both look at the same container?)